app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
```

Comment:
* As stated in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=Build%20your%20project,a%20corresponding%20index.) you are to do the following Kindly do them 
```
Build your project schema.
Create and run the necessary migration files.
Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
Foreign keys should be included.
Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
All columns that are foreign keys should have a corresponding index.
```